cli/script/command-executor.ts

```diff
@@ -317,11 +357,44 @@ function deploymentHistory(command: cli.IDeploymentHistoryCommand): Promise<void
         })
         .then((deploymentId: string): Promise<Package[]> => {
             throwForInvalidDeploymentId(deploymentId, command.deploymentName, command.appName);
+            storedDeploymentId = deploymentId;
 
             return sdk.getPackageHistory(storedAppId, deploymentId);
         })
-        .then((packageHistory: Package[]): void => {
-            printDeploymentHistory(command, packageHistory);
+        .then((packageHistory: Package[]): Promise<void> => {
+            return sdk.getDeploymentMetrics(storedAppId, storedDeploymentId)
```

Comment:
Seeing as we now have this information, would it be worth displaying the binary app versions in the history as well? I'm not strongly opinionated on this point though, I'm mainly concerned with the round-tripping.